contracts/meta-oracles/data-sources/LinearizedEMATimeSeriesFeed.sol

```diff
@@ -21,12 +21,11 @@ import { SafeMath } from "openzeppelin-solidity/contracts/math/SafeMath.sol";
 import { TimeLockUpgrade } from "set-protocol-contracts/contracts/lib/TimeLockUpgrade.sol";
 
```

Comment:
Technically, this isn't a data source anymore. So let's place it into `feed/LinearizedEMATimeSeriesFeed.sol`?